tst/standard/examples.tst

```diff
@@ -226,6 +226,18 @@ Error, the arguments <n> and <k> must be non-negative integers,
 gap> JohnsonDigraph(IsMutableDigraph, 4, 2);
 <mutable digraph with 6 vertices, 24 edges>
 
+# PancakeGraph
+gap> D := PancakeGraph(3);
```

Comment:
Please also add a test for `PancakeGraph(1)` once its behaviour is agreed.

Reply:
Thanks for the comments @wilfwilson !